### PR TITLE
Add support for categories

### PR DESCRIPTION
**Pull request by appacademy-jchurchill**
This PR adds support for categories.

### CODE REVIEW

File: ToDoList/ToDoList/Models/ToDoItemData.cs [hunk 1]
```diff
@@ -10,23 +10,42 @@ namespace ToDoList.Models
...
         /// <returns></returns>
         public static List<ToDoItem> GetToDoItems()
         {
             return Items;
         }
 
-        public static void AddItem(string description)
+        public static List<string> GetCategories()
+        {
+            if (Categories.Count == 0)
```
Comment (jasonluX13) [inline ToDoList/ToDoList/Models/ToDoItemData.cs:29]: Good to have defaults

**Review by jasonluX13 — APPROVED** [review]

File: ToDoList/ToDoList/Models/ToDoItemData.cs [hunk 2]
```diff
@@ -10,23 +10,42 @@ namespace ToDoList.Models
...
+            {
+                Categories.Add("Personal");
+                Categories.Add("Work");
+                Categories.Add("Chores");
+            }
```
Comment (cvera-tech) [inline ToDoList/ToDoList/Models/ToDoItemData.cs:35]: Perhaps we could add an "miscellaneous" category as a catch-all, or even as the default category

**Review by cvera-tech — APPROVED** [review]
Pretty straightforward additions. I am eager to switch from static collections to an actual database.

File: ToDoList/ToDoList/Models/ToDoItemData.cs (hunk 2 above)
Comment (appacademy-jchurchill) [reply ToDoList/ToDoList/Models/ToDoItemData.cs:35]: Good call... I just added another commit that adds a "Misc" category.

**Review by wjlawlor — COMMENTED** [review]
I looked at it.

**Review by wjlawlor — COMMENTED** [review]
I think this will show five checked now?

**Pull request merged**